osquery/tables/system/windows/certificates.cpp

```diff
@@ -323,6 +551,29 @@ void enumerateCertStore(const HCERTSTORE& certStore,
 
   if (certContext == nullptr && GetLastError() == CRYPT_E_NOT_FOUND) {
     TLOG << "    Store was empty.";
+
+    // Personal stores for other users come back as empty, even if they are not.
+    if (storeName == "Personal" && !username.empty()) {
+      // Avoid duplicate rows for personal certs we've already inserted up
+      // front.
+      if (storeLocation != "Users" || boost::ends_with(storeId, "_Classes")) {
+        TLOG << "    Trying harder to get Personal store.";
+
+        // TODO: This can technically be optimized. In certain cases, we will
```

Comment:
Do you mind moving the content of this TODO into an issue and replacing this with
```
// TODO(#1234): Title of the issue
```